text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,289 @@
+  burden (subjective, reviewers' judgment).
+
+## Orthogonal but related non-proposals
+
+The following principles are assumptions I'm making that we currently follow
+and that we should continue to strive for, no matter what browser support
+policy we end up with:
+
+- Follow best practices for accessibilty, fix bug reports from blind users,
+  reach out to blind users in the community about how the accessibility of the
+  web content could be improved.
+- Follow best practices for colorblindness, such as have information conveyed
+  through color also conveyed through an icon or text.
+- Follow best practices for making content usable from mobile devices with a
+  variety of screen sizes.
+
+Please comment if you think any of these should **not** be assumed, but rest
+assured it is not the intent of this RFC to get rid of these kinds of support.
+
+# Relevant data
+
+[CanIUse.com][] has some statistics on global usage of browsers and versions,
+but our audience (developers) isn't the same as the general public.
+
+[CanIUse.com]: http://caniuse.com/usage-table
+
+## Google analytics browser usage stats
+
+We have Google Analytics on crates.io and on rust-lang.org. The entire data set
+of the usage stats by browser, browser verison, and OS are available [in this
+Google sheet][all-data] for the visits to crates.io in the last month. I chose
+to use just crates.io because on initial analysis, the top 90% of visits to
+rust-lang.org were less varied than the top 90% of visits to crates.io.
+
+[all-data]: https://docs.google.com/spreadsheets/d/1qgszm-_-Tn8FLi2v3vicuvyct3Grzz74JWcroILRq8s/edit?usp=sharing
+
+This data does not include those users who block Google Analytics.
+
+This is the top 80% aggregated by browser and major browser version:
+
+| Browser         | Browser Version | Sessions | % of sessions | Cumulative % |
+|-----------------|-----------------|----------|---------------|--------------|
+| Chrome          | 57              | 18,040   | 34.85         | 34.85        |
+| Firefox         | 52              | 8,136    | 15.72         | 50.56        |
+| Chrome          | 56              | 7302     | 14.11         | 64.67        |
+| Safari          | 10.1 (macos)    | 1,592    | 3.08          | 67.74        |
+| Safari          | 10 (ios)        | 1,437    | 2.78          | 70.52        |
+| Safari          | 10.0.3 (macos)  | 851      | 1.64          | 72.16        |
+| Firefox         | 53              | 767      | 1.48          | 73.65        |
+| Chrome          | 55              | 717      | 1.39          | 75.03        |
+| Firefox         | 45              | 693      | 1.34          | 76.37        |
+| UC Browser      | 11              | 530      | 1.02          | 77.40        |
+| Chrome          | 58              | 520      | 1.00          | 78.40        |
+| Safari (in-app) | (not set) (ios) | 500      | 0.97          | 79.37        |
+| Firefox         | 54              | 472      | 0.91          | 80.28        |
```

Comment:
[Caniuse](http://caniuse.com/usage-table) has global browser usage stats, but I still think there's going to be large differences between the general population and a population of developers. I'm not sure where to find browser stats for developers rather than browser stats for rust developers; I'll do some investigating but if anyone knows of any resources I'd love to hear about them!